packages/playwright/src/generators/init/schema.json

```diff
@@ -14,39 +14,7 @@
                 "$source": "projectName"
             },
             "x-prompt": "What is the name of the project for this component?"
-        },
-        "accessibility": {
-            "type": "boolean",
-            "description": "Scaffolding for accessibility testing.",
-            "alias": "a",
-            "x-prompt": "Would you like to install the scaffolding for accessibility testing?",
-            "default": false
-        },
-        "visualRegression": {
-            "type": "string",
-            "description": "Method used to conduct visual regression testing",
-            "alias": "v",
-            "default": "none",
-            "enum": ["none", "native", "applitools"],
-            "x-prompt": {
-                "message": "What type of visual regression tests would you like to use?",
-                "type": "list",
-                "items": [
-                    {
-                        "value": "none",
-                        "label": "None"
-                    },
-                    {
-                        "value": "native",
-                        "label": "Native"
-                    },
-                    {
-                        "value": "applitools",
-                        "label": "Applitools"
-                    }
-                ]
-            }
```

Comment:
Is it possible to retain these questions when running playwright:init? They make for a good user experience having these questions asked, does it add a lot of complexity?

Reply:
Although I suppose it would mean supporting these questions in two places, here and each respective generator....so may not be plausible

Reply:
Yeah so we had a chat with Nandi as well, figured it made sense to just run playwright init with create workspace and only ask the question of the test runner. Like you say, otherwise we would be asking and doing the same things in more than one place. So as it stands, once create workspace is completed and you have selected playwright as test runner, you will get the m&s stuff with the option to run accessibility and/or visualRegression as separate generators

Reply:
Cool, happy with that